index.rst

```diff
@@ -345,6 +370,14 @@ These are the strings to search:
 
 - **config deprecations**: ``ack -B 3 --python "^\s+deprecated=" stack/``
 
+Official Release Documentation
+""""""""""""""""""""""""""""""
+
+Since the pipelines_lsst_io repository is devoted to document automatically
```

Comment:
I wanted here to clarify, that the steps:
- publish pipelines.lsst.io/v/888.0.0 
- merge the 888.0.x pipelines_lsst_io branch into master
are made by the documentation engineer.

Since this is something you do usually, and I don't have visibility on the technical details, please let me know how to phrase this text. 
I also think, we should not give much technical detail, but this is up to you.